### ISSUE TITLE
Bug when Hitting "Back" at Root and then Selecting a Thumbnail 

### ISSUE THREAD

**Issue by FerchoCarcho** [UniSharp/laravel-filemanager#91]
Everything Works..
Just if you are at the Base Root with the 

```
 'allow_multi_user'      => true,
```

You Hit the Back button, 
then You select the Thumbnail.
and then Hit Ckeditor's OK .
You get

```
<p><img alt="" src="/photos//5775a01832e28.jpeg" /></p>
```

no user /1/ Directory selected because the `working_dir` value becomes ' '

```
'user_field'            => 'id',
```

Please Check it. 
Thank you !!!

**Comment by FerchoCarcho** [comment]
Solution would be to modify 
the function
```
function getFileUrl(file) {
```
with
```
if (path =='')
path=home_dir
```
?????

**Comment by g0110280** [comment]
Fixed in 0763f14.
